jax/experimental/jax2tf/jax2tf.py

```diff
@@ -904,13 +908,66 @@ def _reduce_window(jax_f, reducer, init_val, operand, window_dimensions,
                             window_dilations=window_dilation, padding=padding)
   out.set_shape(out_shape)
   return out
+
+def _get_max_identity(dtype):
+  jax_dtype = to_jax_dtype(dtype)
+  val = None
+  if dtypes.issubdtype(jax_dtype, np.inexact):
+    val = -np.inf
+  elif dtypes.issubdtype(jax_dtype, np.integer):
+    val = dtypes.iinfo(jax_dtype).min
+  elif dtypes.issubdtype(jax_dtype, np.bool_):
+    val = False
+  return _safe_convert_to_tensor(val, dtype)
+
+def _get_min_identity(dtype):
+  jax_dtype = to_jax_dtype(dtype)
+  val = None
+  if dtypes.issubdtype(jax_dtype, np.inexact):
+    val = np.inf
+  elif dtypes.issubdtype(jax_dtype, np.integer):
+    val = dtypes.iinfo(jax_dtype).max
+  elif dtypes.issubdtype(jax_dtype, np.bool_):
+    val = True
+  return _safe_convert_to_tensor(val, dtype)
+
+def _reduce_window(operand, init_value, *, jaxpr, consts, window_dimensions,
+                   window_strides, padding, base_dilation, window_dilation):
+  """TensorFlow implementation of reduce_window."""
+  assert len(consts) == 0, "Reduction computation cannot have constants"
+
+  out_shape = _reduce_window_shape(lax._reduce_window_sum, operand, window_dimensions,
+                                   window_strides, padding, base_dilation,
+                                   window_dilation)
+
+  def reducer(arg1: TfVal, arg2: TfVal) -> TfVal:
+    typed_jaxpr = _mk_typed_jaxpr(jaxpr, consts)
+    res, = _interpret_jaxpr(typed_jaxpr, arg1, arg2)
+    return res
+
+  o_spec = tf.TensorSpec(operand.shape, dtype=operand.dtype)
```

Comment:
It seems that everything below here can be shared with _Specialized_reduce_window?

Reply:
Addressed in `54ac20e4`